LoRaEngine/test/LoRaWan.SimulatedTest/LoRaWan.SimulatedTest.csproj

```diff
@@ -35,5 +38,11 @@
     <CodeAnalysisRuleSet>../../../stylecop.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <Import Project="../../../stylecop.props" />
+  <ItemGroup>
+    <PackageReference Update="StyleCop.Analyzers" Version="1.1.118">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
```

Comment:
We should have those in the stylecop.props file, this way they are shared among all projects.